### ISSUE TITLE
Merge this code back into pelias/api

### ISSUE THREAD

**Issue by orangejulius** [pelias/query#86]
A few years back we split this code out, I think to set up for the possibility that more areas of our code would make calls to Elasticsearch besides just the API.

This hasn't panned out, and now there's a bit of extra friction to changing our query code, since we have to make changes in this repo, and then update the version in the API.

For development, it introduces a lot of extra work, especially for a Docker based approach, but also locally, as running `npm link` correctly is at best a timewaster, and easy to get wrong.

Unless we have a really good reason I'm forgetting, I think we should move all this code back into https://github.com/pelias/api. It would be one less package/repo to manage.

@missinglink, your thoughts?

**Comment by missinglink** [comment]
Sounds fair, I think there is value to having all the query generation code in one place, but I'm happy if that place is a directory inside `pelias/api`.

**Comment by orangejulius** [comment]
Yeah that's the thing, there's some query views still in the API :)
